administration/src/util/uint8ArrayToPaddedBase64.ts

```diff
@@ -1,9 +1,9 @@
-const uint8ArrayToBase64 = function (u8: Uint8Array) {
+const uint8ArrayToPaddedBase64 = function (u8: Uint8Array) {
```

Comment:
This function already exists in the `base64.ts` file. I think we can omit `padded` here. It's the default everywhere.

Reply:
Ah sorry, it isn't there (yet). I changed that in my pr #787 ... Still, I think we can drop `padded` :D